File: VB/MVVMExpenses/Views/Account/AccountsEditFormView.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Linq
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports MVVMExpenses.ViewModels
Imports DevExpress.XtraGrid.Views

Namespace MVVMExpenses.Common.Views.Account
    <DevExpress.Utils.MVVM.UI.ViewType("AccountView")>
    Partial Public Class AccountsEditFormView
        Inherits DevExpress.XtraEditors.XtraUserControl

        Public Sub New()
            InitializeComponent()
            If Not DesignMode Then
                InitBindings()
            End If
        End Sub

        Private Sub InitBindings()
            Dim fluent = mvvmContext1.OfType(Of AccountViewModel)()
            fluent.SetObjectDataSourceBinding(accountBindingSource, Function(x) x.Entity, Sub(x) x.Update())
            fluent.SetBinding(GridControl1, Function(gc) gc.DataSource, Function(x) x.AccountTransactionDetails.Entities)
            CType(GridControl1.MainView, DevExpress.XtraGrid.Views.Grid.GridView).Columns("Account").Visible = False
        End Sub


    End Class
End Namespace

File: CS/MVVMExpenses/Views/Account/AccountsEditFormView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Linq;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using MVVMExpenses.Models;
using MVVMExpenses.Models.ViewModels;
using DevExpress.XtraGrid.Views.Grid;

namespace MVVMExpenses.Common.Views.Account {
    [DevExpress.Utils.MVVM.UI.ViewType("AccountView")]
    public partial class AccountsEditFormView : DevExpress.XtraEditors.XtraUserControl {
        public AccountsEditFormView() {
            InitializeComponent();
            if(!DesignMode) InitBindings();
        }

        void InitBindings() {
            var fluent = mvvmContext1.OfType<AccountViewModel>();
            fluent.SetObjectDataSourceBinding(
                accountBindingSource, x => x.Entity, x => x.Update());

            fluent.SetBinding(
                gridControl1, gc => gc.DataSource, x => x.AccountTransactionDetails.Entities);

            ((GridView)gridControl1.MainView).OptionsBehavior.Editable = false;
            ((GridView)gridControl1.MainView).Columns["Account"].Visible = false;
        }
    }
}
